### ISSUE TITLE
404 errors

### ISSUE THREAD

**Issue by rowan-ardc** [SISS/sissvoc#11]
Deployed docker container: docker run -p "8080:8080" csiroenvinf/sissvoc:latest

From /sissvoc/default/concept I see 404 errors from clicking on 'json', 'rdf', 'text', 'ttl', 'xml', 'first', 'next', as well as items within the 'sort by', 'view' and 'items per page' panes.

For example: /default/concept.html?_page=0&_sort=prefLabel

> HTTP Status 404 – Not Found
> 
> Type Status Report
> 
> Message /default/concept.html
> 
> Description The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.
> Apache Tomcat/8.5.38

**Comment by jyucsiro** [comment]
Thanks for that feedback @rowan-ardc 

Seems like the webapp prefix wasn't configured properly. Will take a look.

**Comment by jyucsiro** [comment]
PR #12 aims to address this.

**Comment by jyucsiro** [comment]
I've made changes now and released a new version (https://github.com/SISS/SISSVoc/releases/tag/sissvoc-3.6-rc-2).

The relevant docker images have now been updated. You might need to execute:
```
$ docker pull csiroenvinf/sissvoc:latest
or
$ docker pull csiroenvinf/sissvoc:alpine
```
Then run ` docker run -p "8080:8080" csiroenvinf/sissvoc:latest` or ` docker run -p "8080:8080" csiroenvinf/sissvoc:alpine`

The links should be resolving now.
